src/components/Contact/Chat.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
```

Comment:
Chat.js: Any particular reason for the change from `import React, { Component } from 'react';` to `import React, { Component } from "react";`?